.env.example

```
@@ -1,246 +1,843 @@
+# On Fly deploy: Discord bot machine.
+REALTIME_VAD_INTERRUPT_RESPONSE=
+
+# Optional semantic VAD eagerness (low|medium|high|auto). Leave unset to use provider defaults.
+# `low`, `medium`, `high`, `auto`
+# On Fly deploy: Discord bot machine.
+REALTIME_VAD_EAGERNESS=
+
+# Greeting text sent when a realtime voice session starts. Supports {bot} placeholder.
+# string
+# On Fly deploy: Discord bot machine.
+REALTIME_GREETING=Hey, {bot} here.
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)